sdk/storage/azblob/sas/service.go

```diff
@@ -322,25 +333,29 @@ func (p *ContainerPermissions) String() string {
 			p.DeletePreviousVersion = true
 		case 'l':
 			p.List = true
-		case 't':
+		case 'f':
```

Comment:
Order of these fields matter in the sas, can we put some test cases around the valid and invalid sequences.

Reply:
Yep, This would be validated by the String tests and the tests that generate and use sas in tests